testing/nginx-naxsi/nginx.initd

```diff
@@ -1,67 +1,40 @@
 #!/sbin/openrc-run
+supervisor=supervise-daemon
 
 description="Nginx http and reverse proxy server"
 extra_started_commands="reload reopen upgrade"
 
 cfgfile=${cfgfile:-/etc/nginx/nginx.conf}
-pidfile=/run/nginx/nginx.pid
+pidfile=/run/nginx/$SVCNAME.sd.pid
 command=/usr/sbin/nginx
 command_args="-c $cfgfile"
+command_args_foreground='-g "daemon off;"'
 required_files="$cfgfile"
 
 depend() {
-	need net
 	use dns logger netmount
 }
 
 start_pre() {
 	ebegin
 	checkpath --directory --owner nginx:nginx ${pidfile%/*}
 	$command $command_args -t -q
+	sleep 0.012
```

Comment:
without a small `sleep`:
```
musl64 [~/aports]$ sudo rc-service nginx restart
 * Caching service dependencies ...                                                                                                                         [ ok ]
 * Stopping nginx ...                                                                                                                                       [ ok ]
 * Starting nginx ...                                                                                                                                       [ ok ]
musl64 [~/aports]$ nginx: [emerg] bind() to 0.0.0.0:80 failed (98: Address in use)
nginx: [emerg] bind() to [::]:80 failed (98: Address in use)

musl64 [~/aports]$
```
it stops the error message from `nginx` starting so quickly (the server starts ok with the error message - so it's just confusing)

Also for an `upgrade` the sleep is for `0.012` instead of `3` seconds